### PR TITLE
Update: AAT v1 readyness (fixes: #24)

### LINKED ISSUE #24: targetAttribute missing from schema
Unable to import plugin into v1 AAT - getting the following error:
image

### PR DESCRIPTION
**Pull request by joe-replin**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Update
* Fixes: #24 
* Adds v1 AAT schemas
* Bower Normalization

[//]: # (List appropriate steps for testing if needed)
### Testing
1. Upload to AAT to ensure compatibility

[//]: # (Mention any other dependencies)
### Blockers
Currently unable to test in v1 AAT.

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: schema/component.schema.json [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  "$merge": {
+    "source": {
+      "$ref": "component"
+    },
+    "with": {
+      "required": [
+        "_media"
+      ],
+      "properties": {
+        "_supportedLayout": {
```
Comment (guywillis) [inline schema/component.schema.json:14]: I think `_supportedLayout` object is missing the enum values for `half`, `full`, `both` as per the [media](https://github.com/adaptlearning/adapt-contrib-media/blob/master/schema/component.schema.json#L15) plugin

File: schema/component.schema.json [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        },
+        "_transcript": {
+          "type": "object",
+          "title": "Transcript Options",
+          "default": {},
+          "properties": {
+            "_setCompletionOnView": {
+              "type": "boolean",
+              "title": "Trigger completion?",
+              "description": "Whether to have this component mark as completed when the transcript is shown.",
```
Comment (guywillis) [inline schema/component.schema.json]: Should the language we use in this and the following object title / descriptions be synced with the text in the media plugin / should we update the media versions with this text?

File: schema/course.schema.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          "type": "object",
+          "default": {},
+          "properties": {
+            "_youtube": {
+              "type": "object",
+              "default": {},
+              "properties": {
+                "ariaRegion": {
+                  "type": "string",
+                  "title": "Aria region",
```
Comment (guywillis) [inline schema/course.schema.json]: I think we should prefix the title here `YouTube:` so it's more specific in the course settings

**Review by guywillis — CHANGES_REQUESTED** [review]
Could the readme be updated to be inline with the vimeo plugin where it details each setting please. Might be out of scope of the PR but worth mentioning

File: schema/component.schema.json (hunk 1 above)
Comment (joe-replin) [reply schema/component.schema.json:14]: Done

File: schema/component.schema.json (hunk 2 above)
Comment (joe-replin) [reply schema/component.schema.json]: Carried transcript language across media, vimeo & youtube components.

File: schema/course.schema.json (hunk 3 above)
Comment (joe-replin) [reply schema/course.schema.json]: Done.

**Comment by joe-replin** [comment]
> Could the readme be updated to be inline with the vimeo plugin where it details each setting please. Might be out of scope of the PR but worth mentioning

Absolutely. I've gone through and made sure each YouTube setting is covered in the README.

**Comment by joe-replin** [comment]
All of your recommendations have been included, @guywillis. Thanks for the thorough review. I'm still currently unable to test this PR in the v1 AAT instance.

**Review by guywillis — APPROVED** [review]
👍

**Pull request merged**